compiler-utils/build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id 'software.amazon.lib.jvm'
+    alias(libs.plugins.ksp)
+}
+
+test {
+    useJUnitPlatform()
+
+    maxHeapSize = "2g"
+}
```

Comment:
This file was copied, e.g .this configuration nor KSP are needed in this module.